packages/server/src/tests/utilities/TestConfiguration.ts

```diff
@@ -358,9 +354,9 @@ export default class TestConfiguration {
     } = {}
   ): Promise<User> {
     let { id, firstName, lastName, email, builder, admin, roles } = user
-    firstName = firstName || this.defaultUserValues.firstName
-    lastName = lastName || this.defaultUserValues.lastName
-    email = email || this.defaultUserValues.email
+    ;(firstName = firstName || generator.first()),
+      (lastName = lastName || generator.last()),
+      (email = email || generator.email())
```

Comment:
This looks super weird. It would probably make sense to just have multiple `let`s but I'm not all that bothered if you want to leave it as is.

Reply:
Not sure how this even happened 😓 probably some autoformatter after the replaces

Reply:
https://github.com/Budibase/budibase/pull/13096